CMakeLists.txt

```diff
@@ -13,9 +20,16 @@ else()
 endif()
 find_package (32BLIT CONFIG REQUIRED PATHS ../32blit-sdk)
 
+if(PICO_SDK_PATH)
```

Comment:
... which makes this not a great way to check for a pico build. The blit SDK uses `32BLIT_PICO` for this.